quipucords/tests/utils/raw_facts_generator.py

```diff
@@ -31,8 +31,20 @@ def fake_rhel():
     return f"Red Hat Enterprise Linux release {fake_major_minor_ver()} ({codename})"
 
 
+def fake_installed_products() -> list[dict]:
+    """Return a list representing at least one found installed product."""
+    installed_products = [
+        {
+            "id": _faker.pyint(min_value=100, max_value=999),
```

Comment:
Satellite plugin submits product ids as strings :shrug:   I think they just happen to use numerical characters, but they are strings for all intents and purposes.

Reply:
Oh, I think you're right. I remember reading strings in the spec (https://github.com/RedHatInsights/insights-host-inventory/blob/986a8323f6d5d94ad721a9746cd50f383dd2594c/swagger/system_profile.spec.yaml#L87), and then I promptly forgot to cast the IDs as strings in my test fixtures. I may review and update this to return strings.

Reply:
Fixed! Thanks for catching this.